Dockerfile

```diff
@@ -1,16 +1,33 @@
-FROM ubuntu:22.04
+FROM ubuntu:22.04 AS builder
```

Comment:
@wafflespeanut suggested we should use 20.04 since the latest LTS version have compatibility issues.

Reply:
Let's open another PR for that

Reply:
We can also use `debian-11`? Can you open an issue for this.

Reply:
https://github.com/DeFiCh/metachain/issues/166